resources/views/requirements.blade.php

```diff
@@ -280,7 +280,7 @@
         </div>
     </div>
 
-    @yield('entry_btn')
+    @include('components.entry_btn',['entryGtm' => {{requirements_entry_01}},'requirementsGtm' => {{requirements_requirements_01}}])
```

Comment:
@yosuke-fujita 
```suggestion
    @include('components.entry_btn',['entryGtm' => 'requirements_entry_01','requirementsGtm' => 'requirements_requirements_01'])
```
私の指示書の書き方が紛らわしくてすみません…！
上のように波括弧は消して`'`で囲って欲しいです！